content/articles/2017-11-01-lubridate-1.7.0.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+title: lubridate 1.7.0
+slug: lubridate-1.7.0
+description: >
+  A new version of lubridate is on CRAN!
+author: Vitalie Spinu
+date: 2017-11-01
+photo:
+  url: https://unsplash.com/photos/yBzrPGLjMQw
+  author: Heather Zabriskie
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library("lubridate")
+knitr::opts_chunk$set(collapse = TRUE, comment = "##=>")
+```
+
+We are pleased to announce that `lubridate` 1.7.0 is now on CRAN! For a complete
```

Comment:
* Just realised this needs a brief description of lubridate for those who aren't familiar with it. e.g. lubridate makes working with dates and times less painful by X, Y, and Z.

* And house style is format package names as regular names (i.e. no backticks)

Once that's done, I'll merge.

Reply:
> Just realised this needs a brief description of lubridate

I have added a link to CRAN's lubridate page. That should be enough. Lubridate is among the oldest packages in tidyverse, people surely heard about it. It's an announcement  of a new version targeted for people who use it, not an introduction of some sort. None of the earlier articles have added such descriptions either.

>  (i.e. no backticks)

Fixed.

Reply:
There are lots of people new to the tidyverse and I think its a really good idea to include a very brief description. If previous blog posts didn't do it, it's only because I missed it.

Reply:
>  really good idea to include a very brief description.

Ok, I disagree, but adding it because it's your house and your rules ;-) I believe such post should be brief and to the point, excessive verbosity is distractive. One-click-away-description is a perfect trade-off IMO. 

BTW, I am considering rewriting lubridate from scratch and extracting the parser with some major simplifications. Hopefully within 6 months, but no promises.